metricbeat/docs/developer-guide/index.asciidoc

```diff
@@ -9,18 +8,20 @@
 Metricbeat is periodically interrogating other services to fetch key metrics
 information. As a developer Metricbeat can be used in two different ways:
 
-* As application to extend
-* As library
+* Extend Metricbeat directly
+* Create your own beat and use Metricbeat as a library
 
-That means as a contributor you have different options on how to use metricbeat
-and how to contribute.
+We recommend to start with your own beat as it keeps the development of your own module / metricset indepent.
+If it should be added a later stage as a module in Metricbeat, the same code can be reused and not additional
+changes are needed.
 
 If you are looking to contribute in general to beats or the foundation of
 metricbeat, also have a look at the
 https://github.com/elastic/beats/blob/master/CONTRIBUTING.md[general contributor guide].
 
 --
 
+
 == Overview
```

Comment:
@dedemorton I tried to add [float] here as I want all this on one page, but it breaks the build. Is this because of pageintro?